hedera-node/hedera-addressbook-service-impl/build.gradle.kts

```diff
@@ -32,7 +32,6 @@ mainModuleInfo { annotationProcessor("dagger.compiler") }
 testModuleInfo {
     requires("com.hedera.node.app.service.addressbook.impl")
     requires("com.hedera.node.app.service.mono")
```

Comment:
Is it possible to remove the dependency on `com.hedera.node.app.service.mono`?

Reply:
I can check not sure
I didn't add it